core/server/master/src/main/java/alluxio/master/block/meta/WorkerMeta.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.master.block.meta;
+
+import alluxio.util.CommonUtils;
+import alluxio.wire.WorkerNetAddress;
+
+import com.google.common.base.Preconditions;
+import net.jcip.annotations.ThreadSafe;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+/**
+ * An object representation of the worker metadata.
+ * This class is thread safe so accessing or updating the fields do not require locking.
+ * The only mutable field is the last updated timestamp.
+ */
+@ThreadSafe
+public class WorkerMeta {
+  /** Worker's address. */
+  final WorkerNetAddress mWorkerAddress;
+  /** The id of the worker. */
+  final long mId;
+  /** Start time of the worker in ms. */
+  final long mStartTimeMs;
+  /** Worker's last updated time in ms. */
+  final AtomicLong mLastUpdatedTimeMs;
```

Comment:
Would this fit better to `MasterWorkerInfo` since it doesn't require lock like its neighbors?

Reply:
Moved into `MasterWorkerInfo`